typings.d.ts

```diff
@@ -0,0 +1,67 @@
+interface EntitySchemaOptions {
```

Comment:
All types that are used by public APIs should be exported? So I should export everything in this file, right?